src/python/doc/ripsnet.rst

```diff
@@ -0,0 +1,52 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+RipsNet user manual
+=========================
+Definition
+----------
+
+.. include:: ripsnet.inc
+
+:class:`~gudhi.ripsnet` constructs a Tensorflow model for fast and robust estimation of persistent homology of
+point clouds.
+RipsNet is based on a `Deep Sets  <https://papers.nips.cc/paper/2017/file/f22e4747da1aa27e363d86d40ff442fe-Paper.pdf>`_
+architecture, for details see `RipsNet  <https://arxiv.org/abs/2202.01725>`_.
```

Comment:
I would say that the use of `biblio/bibliography.bib` (then `:cite:` tag) is preferred to links for scientific articles.

Reply:
I tried this and added entries to the bibliography file but I think something is wrong/missing. It does not seem to work properly. As soon as this works the previous links can be removed.